src/main/java/steps/AndroidLoginSteps.java

```diff
@@ -0,0 +1,80 @@
+package steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import skynet.interactions.Elements;
+import skynet.interactions.Wait;
+import skynet.utils.AHCustomAsserts;
+import skynet.utils.Logger;
+
+public class AndroidLoginSteps {
+    @When("user tap on the Login text button")
+    public void tapOnLoginButton() throws Exception {
+        Logger.info("tap Login text button");
+        WebElement el = Elements.findElement(By.id("textViewLogin"));
+        el.click();
```

Comment:
Please use the methods from the framework

Reply:
updatede